src/live/handleAlbumChat.js

```diff
@@ -1,11 +1,36 @@
-function handleAlbumChat(io, socket, data) {
-  const { message, userId, albumId } = data;
+const ChatMessage = require('../models/ChatMessage');
 
-  // Create a chat room name for the album chat
-  const chatRoomName = `chat:album:${albumId}`;
+async function handleAlbumChat(io, socket, data) {
+  try {
+    const { albumId } = data;
 
-  // Send the message to the room
-  socket.to(chatRoomName).emit('chat:album', { message, userId, albumId });
+    // Create a chat room name for the album chat
+    const chatRoomName = `chat:album:${albumId}`;
+
+    const msg = await ChatMessage.create({ ...data, name: data.user });
```

Comment:
`data.user` that the frontend is passing us is a user id as far as I see in the frontend code: [https://github.com/Code-the-Dream-School/dd-prac-team4-front/pull/52/files#diff-dc044c6ce14501135532d751cb2c59b15079b67646f1634abe6dd9c739c899b4R26](https://github.com/Code-the-Dream-School/dd-prac-team4-front/pull/52/files#diff-dc044c6ce14501135532d751cb2c59b15079b67646f1634abe6dd9c739c899b4R26)

Also the `ChatMessage` schema does not have a `name` attribute so this field will just be ignored. We can keep it as you had before:

```suggestion
    const msg = await ChatMessage.create(data);
```